_data/shared_hosts.yml

```diff
@@ -156,10 +156,10 @@
 
 - name: "names.co.uk"
   url: "https://www.names.co.uk/web-hosting/"
-  php54: 35
-  php55: 19
-  php56: 3
-  default: 5.5.19
+  php54: 36
+  php55: 30
+  php56: 4
+  default: 5.5.20
```

Comment:
Oh yeah 5.5.30 doesn’t exist at all.

This is why we’re working on a automated solution for all this. :) 

-- 
Phil

From: Graham Campbell <notifications@github.com>
Reply: philsturgeon/phpversions.info <reply@reply.github.com>>
Date: January 5, 2015 at 4:48:36 PM
To: philsturgeon/phpversions.info <phpversions.info@noreply.github.com>>
Cc: Phil Sturgeon <me@philsturgeon.uk>>
Subject:  Re: [phpversions.info] Updated version numbers for names.co.uk & Register365 (#64)  

In _data/shared_hosts.yml:

> @@ -156,10 +156,10 @@
>   
>  - name: "names.co.uk"
>    url: "https://www.names.co.uk/web-hosting/"
> -  php54: 35
> -  php55: 19
> -  php56: 3
> -  default: 5.5.19
> +  php54: 36
> +  php55: 30
> +  php56: 4
> +  default: 5.5.20
or just a total typo

—
Reply to this email directly or view it on GitHub.